asynchronous_subprocess.py

```diff
@@ -0,0 +1,55 @@
+import asyncio
+import time
+import websockets
+from websocket_client import PSWebsocketClient as Client
+
+async def main():
```

Comment:
Write some comments about the purpose of this file, e.g. that it is called by env_pkmn_smogon.py to (presumably) spawn a new Pokemon game.